WordPress/Classes/Services/MediaCoordinator.swift

```diff
@@ -77,7 +94,19 @@ class MediaCoordinator: NSObject {
             return
         }
         mediaProgressCoordinator.track(numberOfItems: 1)
-        begin(media)
+        uploadMedia(media)
+    }
+
+    /// Starts the upload of a media object
+    ///
+    /// - Parameter media: the media to upload
+    ///
+    func syncMedia(_ media: Media) {
```

Comment:
I'm not a fan of the name of this method, as we already have another `syncMedia` method that does something totally different. We kind of want it to be `uploadMedia`, but we already have a (private) upload method...

It's still a bit confusing, but we could rename the private one to

`private func upload(_ media: Media) -> Progress`

and this new one to

`func uploadMedia(_ media: Media)`

Any other ideas?

Reply:
Oh! How about another `addMedia`? But it takes a Media object as a parameter. That probably makes the most sense for me, and it just doesn't create a new media object, but does the upload. :)